pyproject.toml

```diff
@@ -374,4 +374,10 @@ ignore = [
 ]
 "utils/_features.py" = [
     "ANN205"  #obvious decorators
-]
\ No newline at end of file
+]
+
+[tool.yamlfix]
+line_length = 9001 # effectively disabled
+comments_whitelines = 0
+comments_min_spaces_from_content = 1
+quote_representation = '"'
```

Comment:
Why this is defined to use double quotes, and we have single quotes in this PR ?

Reply:
It's not great, but it's acceptable?

Reply:
what's the point of having a formatter if it doesn't format correctly, at this point let the people just format however they want, it's yaml, it can't be that difficult